form.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<form action="http://127.0.0.1:5000/api/projects/search/simple/1/1/" method="post">
+  First name:<br>
+  <input type="text" name="st" value="Mickey">
+  <input type="submit" value="Submit">
+</form> 
+
+<p>If you click the "Submit" button, the form-data will be sent to a page called "/action_page.php".</p>
+
+</body>
+</html>
+
```

Comment:
I have not a single clue. I don't know who made it or why it's there.
It's gone now.